lib/mirage.ml

```diff
@@ -688,7 +688,7 @@ let extra_c_artifacts target pkgs =
   in
   R.ok r
 
-let static_libs pkg_config_deps = pkg_config pkg_config_deps [ "--static" ; "--libs" ]
+let static_libs pkg_config_deps = pkg_config pkg_config_deps [ "--static" ; "--libs"]
```

Comment:
Spurious whitespace change.

Reply:
Genode is linked shared, so I added a `shared_libs` function, but it makes no difference, so I manually reverted and the whitespace change snuck in.